### ISSUE TITLE
AttributeError:  'PointCloud' object has no attribute 'sample'

### ISSUE THREAD

**Issue by Aristkj** [mikedh/trimesh#1288]
When i use
“mesh = trimesh.load(folder + "registeredScene.ply")
 mesh_copy = mesh.copy()
 sample_points = mesh_copy.sample(10000)”
Statement，system hint C，How can i modify it？Thanks。

**Comment by mikedh** [comment]
Yeah if the PLY doesn't contain any faces trimesh will load it as a `PointCloud` object, which doesn't have a surface to sample. You could sample it as described here: https://github.com/mikedh/trimesh/issues/820
